resources/grafana/rhacs-central-slo-dashboard.yaml

```diff
@@ -162,7 +162,7 @@ spec:
               },
               "editorMode": "code",
               "exemplar": false,
-              "expr": "count(central:slo:availability:error_budget_exhaustion{rhacs_org_id!=\"16536854\"} <= 1 and central:sli:availability) or vector(0)",
+              "expr": "count(central:slo:availability:error_budget_exhaustion{rhacs_instance_id=~\"$instance_id\"} <= 1 and central:sli:availability) or vector(0)",
```

Comment:
What's `$instance_id` and where is the value coming from? And why this change(and similar changes) here is needed?

Reply:
`$instance_id` is part of the dashboard variables you see on top of the screen. Note that multiple instances may be selected - note that `=~`, which in promql means compare with regex.

<img width="742" alt="Screenshot 2023-07-25 at 00 26 04" src="https://github.com/stackrox/rhacs-observability-resources/assets/55607356/7a71cee9-4563-4133-b663-469c9ec0b6fd">

These variables work like this: You can select a metric, and it gives you all label values of this metric for selection. Previously we used something like `central:sli:availability{org_id!="probe-organisation"}` as the metric.

<img width="1488" alt="Screenshot 2023-07-25 at 00 29 13" src="https://github.com/stackrox/rhacs-observability-resources/assets/55607356/082bfc7e-b7d2-4707-91f3-3e62fe89215b">

Now the major change that was fixed in https://github.com/stackrox/rhacs-observability-resources/pull/112 is that SLI metrics no longer have `rhacs_org_*` labels. That is because some of the SLI now come from HAProxy, which has no knowledge about ACS tenants. As a result `central:sli:availability` and derived metrics also no longer have `rhacs_org_*` labels. We therefore need to change the variable definition to another metric (`process_cpu_seconds_total`). We also need to merge this metric in the table view to get all the columns.